app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt

```diff
@@ -7,6 +7,10 @@ import com.google.android.material.snackbar.Snackbar
 interface BaseFragmentContract {
     fun overrideEnterAnimation(@AnimRes animation: Int)
     fun overrideExitAnimation(@AnimRes animation: Int)
+    fun showToast(
+        message: String? = null,
+        @StringRes messageResId: Int? = null
+    )
```

Comment:
use `TextContainer` instead of two nullable types

Reply:
Waaaat ?

Reply:
You'll be able to pass 
`TextContainer(R.string.some_string)` or 
`TextContainer("Some string")`

And get the value by `getString(context)`

Reply:
ohh i see
Djezd'

Reply:
@1sakov fixed